README.md

```diff
@@ -49,20 +33,23 @@ $ python setup.py install
 
 ## Documentation
 
-The python documentation is built using [sphinx](http://sphinx-doc.org/) and
-[numpydoc](https://pypi.python.org/pypi/numpydoc).
-
 ```
-$ make doc
+from bdbcontrib import Population
+help(Population)
+foo = Population(...)
+foo.help("plot")
 ```
 
-## Use
+## Contributing
 
-If you want the Python API, `import bdbcontrib.plot_utils`, &c.
+This repository is currently using "Light Review" from
+http://tinyurl.com/probcomp-review-standards
 
-If you are using the bayeslite shell, load the bayeslite shell
-commands with:
-
-```
-.hook /path/to/bdbcontrib/hooks/hook_plotting.py
-```
+Our compatibility aim is to work on probcomp machines and members'
+laptops, and to provide scripts and instructions that make it not too
+hard to re-create our environments elsewhere. Polished packaging,
+broad installability without much work, etc. are anti-goals, because
```

Comment:
The problem is the awkward use of the word "anti-goal", which [means](https://en.wiktionary.org/wiki/antigoal): _An undesirable state to be avoided_.

Clearly it is __not__ undesirable for the packaging to be highly polished and installable; it undesirable for us to be _spending time_ doing that, which your current README does not make clear. If you want to use the word "anti-goal" we should be very specific about what exactly is the "anti-goal".